package.json

```diff
@@ -11,11 +11,13 @@
   "scripts": {
     "lint": "tslint src/**/*.ts test/**/*.ts",
     "typings-checker": "typings-checker --allow-expect-error --project typings-checker/tsconfig.json typings-checker/index.ts",
+    "prettier": "prettier --write \"{src,test}/**/*.ts\"",
+    "prettier-check": "prettier --list-different \"{src,test}/**/*.ts\"",
     "mocha": "mocha -r ts-node/register test/*.ts",
-    "test": "npm run lint && npm run typings-checker && npm run mocha",
+    "test": "npm run prettier-check && npm run lint && npm run typings-checker && npm run mocha",
```

Comment:
I'd remove `prettier-check` from here since it's already run in `preversion` anyway